apps/src/lib/script-editor/LevelTokenDetails.jsx

```diff
@@ -183,6 +186,13 @@ const LevelTokenDetails = React.createClass({
             />
           </div>
         )}
+        {/* We don't currently support editing progression names here, but do
+          * show the current progression if we have one. */}
+        {this.props.level.progression &&
+          <div style={styles.progression}>
+            Progression Name: {this.props.level.progression}
+          </div>
+        }
```

Comment:
Do we need to do anything else to make sure this value gets serialized (so edit + save doesn't lose this property)?  I'll take a closer look this morning.